src/utils/aquarius.ts

```diff
@@ -240,3 +249,57 @@ export async function getPublishedAssets(
     }
   }
 }
+
+export async function getDownloadAssets(
+  didList: string[],
+  tokenOrders: OrdersData[],
+  chainIds: number[],
+  cancelToken: CancelToken
+): Promise<DownloadedAsset[]> {
+  const downloadedAssets: DownloadedAsset[] = []
+
+  try {
+    const searchDids = JSON.stringify(didList)
+      .replace(/,/g, ' ')
+      .replace(/"/g, '')
+      .replace(/(\[|\])/g, '')
+      // for whatever reason ddo.id is not searchable, so use ddo.dataToken instead
```

Comment:
this I'm not sure as I discovered this pattern here in our `Bookmarks` component. So worth checking if we can simply search for `ddo.id` by now with all the latest Aquarius changes